planb/cli.py

```diff
@@ -78,8 +82,45 @@ def create(regions: list,
     create_cluster(options=locals())
 
 
-sns_topic_help = 'SNS topic name to send Auto-Recovery notifications to'
-sns_email_help = 'Email address to subscribe to Auto-Recovery SNS topic'
+@cli.command()
+@click.option('--region', type=str, required=True)
+@click.option('--cluster-name', type=str, required=True)
+@click.option('--cluster-size', default=3, type=int, help='number of nodes per region, default: 3')
+@click.option('--dc-suffix', type=str, required=True)
```

Comment:
Shouldn't we validate the input to allow valid chars only? Using of option callbacks seems appropriate... http://click.pocoo.org/5/options/#callbacks-for-validation